admin/c2cgeoportal_admin/views/layer_groups.py

```diff
@@ -16,6 +19,15 @@
 base_schema.add(metadatas_schema_node.clone())
 base_schema.add_unique_validator(LayerGroup.name, LayerGroup.id)
 
+# Used for the creation of a new layer group from the layertree
+base_schema.add(colander.SchemaNode(
+    colander.Integer(),
+    name='parent_id',
+    missing=colander.drop,
+    validator=colander.deferred(partial(base_deferred_parent_id_validator, model=Theme)),
+    widget=deform.widget.HiddenWidget()
+))
+
```

Comment:
Despite of model, this could be factorized amongst layerWmsView, WmtsView, and treeGroupView ? Moved to schema directory ?

Reply:
using something like: 
```
def parent_id_node(model):
    return colander.SchemaNode(
        colander.Integer(),
        ...
        validator=colander.deferred(...., model=model)),
```

Reply:
So i factorize this by creating a new function in the schema?

Reply:
In schemas directory ? c2cgeoportal/admin/c2cgeoportal_admin/schemas/ ?

Reply:
This is related to TreeItem subclasses so I would put it in admin/c2cgeoportal_admin/schemas/treeitem.py, seems that related validator shoud be in same file.

Reply:
this is what i meant. I will do that when I get back to work on monday